doc/template.txt

```diff
@@ -32,11 +32,18 @@ determine which template is used:
 	$ vim foo.c
 <
 Templates can be manually expanded in the current buffer using the
-|:Template| command. For example, the following will load and expand the
-template for a file matching the pattern `*.c`:
+|:Template| command in two ways. 
+
+For example, the following will load and expand the
+template for a file matching the file extension:
+>
+	:Template
+>
```

Comment:
This is fun... I have seen before `<` to close example blocks in the Vim documentation, but what you did here works as well, so let's just keep it :man_shrugging:

Reply:
Sure. Also, there are some other combinations used in the [docs of the vim](https://github.com/vim/vim/blob/master/runtime/doc/).